s/sphinx_rtd_theme/sphinx_rtd_theme_ubi_8.5.sh

```diff
@@ -0,0 +1,48 @@
+# -----------------------------------------------------------------------------
+#
+# Package       : sphinx_rtd_theme
+# Version       : 0.5.1
+# Source repo   : https://github.com/readthedocs/sphinx_rtd_theme
+# Tested on		: UBI 8.5
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Saurabh Gore <Saurabh.Gore@ibm.com> / <smohite@us.ibm.com> 
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+
+#Variables
+PACKAGE_NAME=sphinx_rtd_theme
+PACKAGE_VERSION="${1:-0.5.1}"
+PACKAGE_URL=https://github.com/readthedocs/sphinx_rtd_theme.git
+
+#Install dependencies.
+yum install -y git python2 python2-devel python3 python3-devel gcc-c++ make
```

Comment:
Is `python2` required? I see use of `python3` in script.

Reply:
@Siddhesh-Ghadi  Yes , Python2 is required for building package.

Reply:
Okay